doc/resource/git-history.cache

```diff
@@ -2638,7 +2638,7 @@
     {
         "commit": "410a04a58ee59de3055483a2f971c2e42015267e",
         "date": "2018-11-28 14:20:12 -0500",
-        "subject": "Allow arbitary InOut filters to be chained in IoFilterGroup.",
+        "subject": "Allow arbitrary InOut filters to be chained in IoFilterGroup.",
```

Comment:
This appears to be git commits, which probably can't be rewritten. I can trivially drop changes to this file if requested (assuming it actually cares about the commit hashes)